utils/zerovec/src/varzerovec/serde.rs

```diff
@@ -154,7 +72,7 @@ where
     where
         D: Deserializer<'de>,
     {
-        let visitor: VarZeroVecVisitor<T, F> = VarZeroVecVisitor::default();
+        let visitor = VarZeroVecVisitor::<T, F>::default();
```

Comment:
Is this actually different code 👀?

Reply:
yep, the former still runs some type resolution in picking a deserialize impl. It picked wrong.